isos/bootstrap/bootstrap

```diff
@@ -59,21 +59,21 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
         cp /bin/tether ${MOUNTPOINT}/.tether/tether-bin
 
         # temporary hack for waiting the network
-        while ! ip addr show eno1 | grep " inet "; do sleep 1;done
```

Comment:
Per @hickeng this is a temporary solution until he can get a basic DHCP client built into the tether.